docs/getting-started/testing-tutorial.md

```diff
@@ -50,17 +53,17 @@ Your directory structure should look like this:
 ```console
 ├── ick.toml
 ├── isort.cfg
-├── move_isort_cfg/
-│   ├── move_isort_cfg.py
-│   └── tests/
-│       └── no_isort/
-│           ├── input/
-│           │   └── pyproject.toml
-│           └── output/
-│               └── pyproject.toml
-└── pyproject.toml
+├── move_isort_cfg.py
+├── pyproject.toml
```

Comment:
I don't think this line is right.

Reply:
You mean you don't think that file should be there? I can check.

Reply:
We need this file because we still loop over projects in runner.py: 
```python
    def run(self) -> Iterable[HighLevelResult]:
        for impl in self.iter_rule_impl():
            qualname = impl.rule_config.qualname

            impl.prepare()
            for p in self.projects:
                if impl.rule_config.project_types and p.typ not in impl.rule_config.project_types:
                    LOG.log(VLOG_1, "Skipping run on %s because it is not among %s", p, impl.rule_config.project_types)
                    continue
```
Without this file, there are no projects so nothing will run.  The tutorial doesn't tell people to create that file, so the instructions won't work.  I can add that to the tutorial, or we can decide how to make ick work with no projects.